app/views/shared/dialogs/_dialog_field_drop_down_list.html.haml

```diff
@@ -1,7 +1,12 @@
 - if edit
   - select_values = field.values.collect(&:reverse)
   - selected ||= wf.value(field.name)
-  = select_tag(field.name, options_for_select(select_values, selected), drop_down_options(field, url))
+  - if field.multi_value?
+    = select_tag(field.name,
+                 options_for_select(select_values, wf.value(field.name)),
+                 drop_down_options(field, url).merge(:multiple => true))
+  - else
+    = select_tag(field.name, options_for_select(select_values, selected), drop_down_options(field, url))
```

Comment:
I feel like this logic would be better contained in the `drop_down_options` method in the helper, that way it's not in the view and we can test it.